contracts/PodCastNFT.sol

```diff
@@ -73,42 +80,28 @@ contract PodCastNFT is ERC721URIStorage, Ownable, ConsensusAdminable {
     function getTokenURI(
         uint256 _tokenId,
         string memory _imageURI,
-        string memory _role,
-        string memory _roleInCommunity,
+        string[] memory _roles,
         string memory _point
     ) internal view returns (string memory) {
         string memory _name = "Membership NFT";
-        string memory _description = "The membership card of this Henkaku community represents the contribution of the podcast.\\n\\n"
-        "**Special thanks**\\n\\n"
-        "NFT Design:\\n\\n"
-        "Digital Garage team\\n\\n"
-        "Yukinori Hidaka, Saoti Yamaguchi, Masaaki Tsuji, Yuki Sakai, Yuko Hidaka, Masako Inoue, Nanami Nishio, Ruca Takei, Ryu Hayashi.\\n\\n"
-        "Engineering:\\n\\n"
-        "isbtty, yasek, geeknees";
+        string
+            memory _description = "The membership card of this Henkaku community represents the contribution of the podcast.\\n\\n"
+            "**Special thanks**\\n\\n"
+            "NFT Design:\\n\\n"
+            "Digital Garage team\\n\\n"
+            "Yukinori Hidaka, Saoti Yamaguchi, Masaaki Tsuji, Yuki Sakai, Yuko Hidaka, Masako Inoue, Nanami Nishio, Ruca Takei, Ryu Hayashi.\\n\\n"
+            "Engineering:\\n\\n"
+            "isbtty, yasek, geeknees";
 
         bytes memory _attributes;
-        if (isCommunityMember(_tokenId)) {
-            _attributes = abi.encodePacked(
-                '"attributes": [{"trait_type": "Role", "value": "',
-                _role,
-                '"},',
-                '{"trait_type": "Henkaku Community member", "value": "',
-                _roleInCommunity,
-                '"},',
-                '{"display_type": "number", "trait_type": "Point", "value": "',
-                _point,
-                '"}]'
-            );
-        } else {
-            _attributes = abi.encodePacked(
-                '"attributes": [{"trait_type": "Role", "value": "',
-                _role,
-                '"},',
-                '{"display_type": "number", "trait_type": "Point", "value": "',
-                _point,
-                '"}]'
-            );
-        }
+        // TODO implement roles
+        _attributes = abi.encodePacked(
+            '"attributes": [{"trait_type": "Role", "value": "',
```

Comment:
Do we remove the role trait? Or will you implement it later?

Reply:
implement it later